CHANGELOG.md

```diff
@@ -41,7 +42,8 @@ and includes an additional section for migration notes.
 - These are the new variables added:
   - db_admin_username (defaults to "postgres")
   - db_host_endpoint (Requires a value. Set in terraform.tfvars to your RDS Cluster's endpoint, similar to "PREFIX-cumulus-db.cluster-000000000000.us-west-2.rds.amazonaws.com")
-  - api_gateway_policy_vpc_id
+  - db_name (Defaults to disaster_recovery_PREFIX. If preserving a database from a previous version of Orca, set to disaster_recovery.)
```

Comment:
Since we are using PREFIX, shouldn't it be `PREFIX_disaster_recovery`? or maybe we just simplify it to `PREFIX_orca`?

Reply:
I like that. Switched to PREFIX_orca. I put the PREFIX at the end because I was paranoid about illegal characters in Postgres names, but the few edge cases that covers aren't realistic.